src/scipp/plotting/tools.py

```diff
@@ -45,12 +45,23 @@ def to_bin_edges(x, dim):
         return concat([left, center, right], dim)
 
 
+def get_colormap(name):
+    """
+    Return a matplotlib colormap.
+    """
+    import matplotlib
+    if hasattr(matplotlib, 'colormaps'):
+        # This exists since matplotlib v3.5
+        return matplotlib.colormaps[name]
```

Comment:
Note that `matplotlib.colormaps[name]` doesn't work when `name` is `None`, while `get_cmap` does.
But we should be fine, as I think we are setting the `cmap` name from the config if it is `None`.

Reply:
So should I change it or are you happy with the current state?

Reply:
tests passed, so let's say we are ok. If it breaks for users, they can report it and we will change it then.